packages/gatekeeper/src/helpers/utils.ts

```diff
@@ -9,19 +7,3 @@ export type ErrorCallback = (e: string) => string | void;
 export function errorCallback(error: string) {
   throw new Error(error);
 }
-
-/**
- * Open another window and navigate to the logout URL.
- * @param {string} logoutURL URL string representing the auth server logout URL endpoint.
- * This function takes the approach of opening a new window and navigating to the logout
- * url of the authentication server in order to go around the browser's CORS policy.
- */
-export function logoutFromAuthServer(logoutURL: string) {
```

Comment:
- Do we want to keep this around?
- Is it used anywhere that we know of?
- How will we ensure that the logout function which you will implement for Reveal is re-usable for other OpenSRP projects?

Reply:
> hmmm,.. I could add it here, though IMO I do not feel like it belongs here

I can get behind this if the plan is to have it as an OpenSRP package

Reply:
yeah I can have it as an openSRP package